File: Sum of Unique Elements.cpp
class Solution {
public:
    int sumOfUnique(vector<int>& nums) {
        int ans=0;
        for(int i=0;i<nums.size();i++){
            int c=0;
            for(int j=0;j<nums.size();j++){
                if(nums[i]==nums[j] && i!=j){
                    c++;
                    break;
                }
            }
            if(c==0){
                ans+=nums[i];
            }
        }
        return ans;
        
    }
};

File: Sum of Unique Elements.py
class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        dct={}
        sum1=0
        for i in nums:
            if i in dct:
                dct[i]+=1
            else:
                dct[i]=1
        for i in nums:
            if dct[i]==1:
                sum1+=i
        print(dct)        
        return sum1 